deploy/controller.yaml

```diff
@@ -0,0 +1,75 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: datamon-csi-controller
+  namespace: dev
+spec:
+  serviceName: "datamon-csi-driver"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: datamon-csi-driver
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: datamon-csi-driver
+    spec:
+      serviceAccountName: datamon-csi-controller-sa
+      containers:
+        - name: csi-external-attacher
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-attacher:v0.4.1
+          args:
+            - "--v=4"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-external-provisioner
+          imagePullPolicy: Always
+          image: gcr.io/onec-co/csi-provisioner:v0.3.1
```

Comment:
noticed the provisioner is in the one concern docker repo while the attacher is using the upstream docker repo.. what's the reasoning here?  is provisioner the component that had the spurious logging output?